src/chatlist.rs

```diff
@@ -547,6 +550,82 @@ mod tests {
         assert_eq!(chats.len(), 1);
     }
 
+    #[async_std::test]
+    async fn test_search_single_chat() -> anyhow::Result<()> {
+        let t = TestContext::new_alice().await;
+
+        // receive a one-to-one-message, accept contact request
+        dc_receive_imf(
+            &t,
+            b"From: Bob Authname <bob@example.org>\n\
+                 To: alice@example.com\n\
+                 Subject: foo\n\
+                 Message-ID: <msg1234@example.org>\n\
+                 Chat-Version: 1.0\n\
+                 Date: Sun, 22 Mar 2021 22:37:57 +0000\n\
+                 \n\
+                 hello foo\n",
+            "INBOX",
+            1,
+            false,
+        )
+        .await?;
+
+        let chats = Chatlist::try_load(&t, 0, Some("Bob Authname"), None)
+            .await
+            .unwrap();
```

Comment:
The test returns `Result`, you can replace most of the `unwrap`s by `?`.

Reply:
thanks for the hint, i did copy+paste from non-resutified tests :) i will change that.